pom.xml

```diff
@@ -16,15 +16,16 @@
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<webappDirectory>${project.build.directory}/${project.build.finalName}</webappDirectory>
-		<synapse.version>2020-03-18-543-12775e2</synapse.version>
+		<synapse.version>308.0</synapse.version>
 		<gwtVersion>2.8.2</gwtVersion>
 		<org.springframework.version>4.3.18.RELEASE</org.springframework.version>
 		<guiceVersion>3.0</guiceVersion>
 		<ginVersion>2.1.2</ginVersion>
 		<xstream.version>1.3.1</xstream.version>
-		<schema-to-pojo.version>0.4.1</schema-to-pojo.version>
+		<schema-to-pojo.version>0.4.4</schema-to-pojo.version>
 		<org.json.version>20090211</org.json.version>
-		<jackson.version>1.8.1</jackson.version>
+		<jackson.version>2.9.9</jackson.version>
```

Comment:
My bad for suggesting this version, there seems to be 2.9.10 available that should work too!